@here/harp-mapview/lib/MapView.ts

```diff
@@ -2864,8 +2823,7 @@ export class MapView extends THREE.EventDispatcher {
             !this.m_firstFrameComplete &&
             this.m_initialTextPlacementDone &&
             !this.isDynamicFrame &&
-            (this.m_textElementsRenderer === undefined ||
-                (this.m_textElementsRenderer.ready && !this.m_textElementsRenderer.loading))
```

Comment:
here we only need to check that the text renderer is not loading. We know it's already initialized because that's precondition for this.m_initialTextPlacementDone to be true.